docs/commands/zed.md

```diff
@@ -32,7 +32,7 @@ sidebar_label: zed
 A Zed lake is a cloud-native arrangement of data, optimized for search,
 analytics, ETL, data discovery, and data preparation
 at scale based on data represented in accordance
-with the [Zed data model](../formats/zed.md#zed-data-model).
+with the [Zed data model](../formats/zed.md).
```

Comment:
This one is subtle. When rendered on GitHub, https://github.com/brimdata/zed/blob/main/docs/formats/zed.md#zed-data-model is a valid anchor destination attached to the page title. But on the production site, the rendered page https://zed.brimdata.io/docs/next/formats/zed/, the page title doesn't get an anchor.